app/views/downloads/download_linux.html.erb

```diff
@@ -1,7 +1,15 @@
 <%- @section = "downloads" %>
 <%- @subsection = "" %>
 
-<div id="main">
+<div id="main" class="maincontent">
+  <style>
+    .maincontent pre {
+      height: 10px;
+      overflow-y: hidden;
+      padding-top: 0;
+      margin-top: 0;
+    }
+  </style>
```

Comment:
Sticking a `<style>` tag in the body is a bit unusual. I think @pedrorijo91's comment in https://github.com/git/git-scm.com/pull/1495#pullrequestreview-467037583 meant to put it into `app/assets/stylesheets` (probably in `downloads.css`?).

Reply:
I don't know the layout of the codebase, and there wasn't any ```downloads.css``` inside the ```app/assets/stylesheets``` directory. What should I edit?